eth/tracers/native/prestate.go

```diff
@@ -102,6 +102,9 @@ func (t *prestateTracer) CaptureStart(env *vm.EVM, from common.Address, to commo
 	// The recipient balance includes the value transferred.
 	toBal := new(big.Int).Sub(t.pre[to].Balance, value)
 	t.pre[to].Balance = toBal
+	if env.ChainRules.IsEIP158 && create {
```

Comment:
Noo you don't need to expose ChainRules there is another way of getting it:

```go
	rules := env.ChainConfig().Rules(env.Context.BlockNumber, env.Context.Random != nil, env.Context.Time)
	```

Reply:
Done.